packages/api/src/services/google.calendar.ts

```diff
@@ -44,19 +43,20 @@ export class GoogleCalendarService {
         : EventResponseStatus.NEEDS_ACTION;
     };
 
+    const isAgreeToEvent =
+      extendedProperties?.private?.isAgreeToEvent === "true";
```

Comment:
As far as I remember, this is google event calendar custom payload and it only accepts strings. We inject our parameter so that we can understand which event is created by us.